daml-lf/transaction-scalacheck/src/main/scala/com/digitalasset/daml/lf/value/ValueGenerators.scala

```diff
@@ -166,7 +168,9 @@ object ValueGenerators {
         Gen.zip(genAbsCidV1, arbitrary[Byte]) map {
           case (b1, b) =>
             AbsoluteContractId.V1
-              .assertBuild(b1.discriminator, b1.suffix ++ Bytes.fromByteArray(Array(b)))
+              .assertBuild(
+                b1.discriminator,
+                b1.suffix.slice(0, b1.suffix.length min 93) ++ Bytes.fromByteArray(Array(b)))
```

Comment:
Same with the `93` here.

Reply:
Since the append is now more trouble than it was before, given the aforementioned motivation for doing this at all:

```suggestion
                if (b1.suffix.nonEmpty) b1.suffix else Bytes fromByteArray Array(b)
```

Reply:
Done. Thanks.